src/citrine/resources/status_detail.py

```diff
@@ -22,7 +22,7 @@ class StatusDetail(Serializable[StatusDetailType]):
     """A status message and its level."""
 
     msg = properties.String("msg")
-    level = properties.String("level")
+    level = properties.Enumeration(StatusLevelEnum, "level")
```

Comment:
This is an interface break. We'll need to deprecate it as a `String` and make it an `Enumeration` in 3.0.

Reply:
I don't know if I agree, but it's tangential to the purpose of this PR, so I'm removing it for the sake of expediency.  

(The route a user would have to go down in order to have code break feels like they've already violated our API guarantees.  But formally, yes, formerly valid data is no longer valid, and a user could have called the level setter.)